crates/wallet/src/wallet/tx_builder.rs

```diff
@@ -910,13 +930,28 @@ mod test {
     }
 
     #[test]
-    fn test_output_ordering_bip69() {
+    fn test_output_ordering_custom_but_bip69() {
```

Comment:
👍 Glad you were able to keep this test by reproducing the bip69 ordering with custom functions.